shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.m

```diff
@@ -23,4 +25,85 @@ - (void)testBinaryMessenger {
   OCMVerify([engine binaryMessenger]);
 }
 
+- (void)testItReportsLightPlatformBrightnessByDefault {
+  // Setup test.
+  id engine = OCMClassMock([FlutterEngine class]);
+
+  id settingsChannel = OCMClassMock([FlutterBasicMessageChannel class]);
+  OCMStub([engine settingsChannel]).andReturn(settingsChannel);
+
+  FlutterViewController* vc = [[FlutterViewController alloc] initWithEngine:engine
```

Comment:
Are these tests running with ARC? Because these look like leaks. The rest of the engine does not use ARC.

Reply:
No idea. I was just continuing with what I already found in this file. If you have any specific recommendations for changes, I can make them, but I'm working from a lack of iOS knowledge here.

Reply:
@chinmaygarde @stuartmorgan do either of you have any specific instructions for what should be done here?

Reply:
@dnfield @gspencergoog do either of you know what changes should be made here to avoid leaks? I don't know what changes to make to bring this test code into compliance...

Reply:
I would turn on ARC for the test file, since that would be easy (it looks like the test project is doing it file-by-file, which seems like an odd approach; I would just enable it for the whole project).

Failing that, add an `autorelease` here.

Reply:
I don't know how to turn on ARC and I also don't know what the structure of our testing system is here so that sounds like a risky thing for me to do.

I'll add autorelease if that solves the problem. Do I add it only to the vc references?

Reply:
@LongCatIsLooong helped me get ARC enabled

Reply:
We may want to have ARC off for the project so that we can explicitly test deallocating things...

Reply:
I believe I only enabled it for this file. Do you want me to change something?

Reply:
No, that's fine - my reply was more directed towards @stuartmorgan's comment :)

Reply:
Well since the only test in that file is leaking the view controller, I'm pretty sure it's not deliberately off to test dealloc...